### ISSUE TITLE
Medren and Mensurations missing.

### ISSUE THREAD

**Issue by alexandermorgan** [cuthbertLab/music21#322]
Am I missing something? music21.alpha.medren isn't anywhere in the repository even though it's in the documentation [here](http://web.mit.edu/music21/doc/moduleReference/moduleAlphaMedren.html
).
I was actually looking for music21.alpha.medren.Mensuration, so if Mensuration has just been moved somewhere else, that would be good enough for my case. I realize that you're not maintaining music21.alpha, but its contents should still be accurately described in the docs.

**Comment by mscuthbert** [comment]
Most of the Alpha files were moved at v.4 or v.5 to a separate repository: music21-tools : https://github.com/cuthbertLab/music21-tools/

You'll find the file at:
https://github.com/cuthbertLab/music21-tools/blob/master/trecento/medren.py  

Alpha files were designed to either eventually become a core part of music21 or be moved out.  I've removed ones that ended up having more of a maintenance burden than I saw usage for.  I'll gladly take pull requests for the music21-tools, but I'm not ensuring that they always work or pass all tests

Thanks for reminding me to clean out old docs that don't get automatically deleted on each new releasee.  Done
